lib/node_modules/@stdlib/number/float64/to-json/benchmark/benchmark.js

```diff
@@ -0,0 +1,95 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var PINF = require( '@stdlib/constants/float64/pinf' );
+var NINF = require( '@stdlib/constants/float64/ninf' );
+var pkg = require( './../package.json' ).name;
+var toJson = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg, function benchmark( x ) {
+	var v;
+	var i;
+
+	x.tic();
+	for ( i = 0; i < x.iterations; i++ ) {
+		v = toJson( NaN );
+		if ( typeof v !== 'object' ) {
+			x.fail( 'should return an object' );
+		}
+		v = toJson( PINF );
+		if ( typeof v !== 'object' ) {
+			x.fail( 'should return an object' );
+		}
+		v = toJson( NINF );
+		if ( typeof v !== 'object' ) {
+			x.fail( 'should return an object' );
+		}
+	}
+	x.toc();
+	if ( typeof v !== 'object' ) {
+		x.fail( 'should return an object' );
+	}
+	x.pass( 'benchmark finished' );
+	x.end();
+});
+
+bench( pkg+'::number', function benchmark( x ) {
+	var v;
+	var i;
+
+	x.tic();
+	for ( i = 0; i < x.iterations; i++ ) {
+		v = toJson( 3.14 );
+		if ( typeof v !== 'number' ) {
+			x.fail( 'should return a number' );
+		}
+	}
+	x.toc();
+	if ( typeof v !== 'number' ) {
+		x.fail( 'should return a number' );
+	}
+	x.pass( 'benchmark finished' );
+	x.end();
+});
+
+bench( pkg+'::number,large', function benchmark( x ) {
```

Comment:
You can get rid of this benchmark. The magnitude of a finite number should not affect behavior and hard to imagine why it would.